### ISSUE TITLE
disposeOnUnmount not working on prototype

### ISSUE THREAD

**Issue by jer-sen** [mobxjs/mobx-react#702]
When I use `disposeOnUnmount ` as a decorator on a property of a `React.Component` with React Native, I get the error `Attempting to change configurable attribute of unconfigurable property.`

A workaround is to change this line https://github.com/mobxjs/mobx-react/blob/00a3f94d52f929472c30be6fa6c42fb469d81db6/src/disposeOnUnmount.js#L86 with `configurable: true`.

`
react@16.8.3
react-native@0.59.0
mobx-react-devtools@6.1.1
`

**Comment by mweststrate** [comment]
This might be fixed once #703 is merged

**Comment by mweststrate** [comment]
Please check if 6.1.0 solves the problem.

**Comment by FredyC** [comment]
Closing as it should be fixed. If the problem persist, please open a new issue with reproduction on the latest package version.
